### PR TITLE
[IOSurface] Updates for Xcode13 Beta 1

### PR DESCRIPTION
**Pull request by tj-devel709**
I don't feel too confident with this one.
I added the missing p-invokes to the ignore since they are ignored in other platforms.

I added the two fields. They say they are supported in older versions and the iOS 11.4 intro test did not complain about them, but a good amount of other IOSurface* fields are in the common ignore and I am not sure why.

### CODE REVIEW

File: src/iosurface.cs [hunk 1]
```diff
@@ -228,5 +228,13 @@ interface IOSurface : NSSecureCoding
 		[iOS (11,0)][Mac (10,13)]
 		[Internal, Export ("setPurgeable:oldState:")]
 		int _SetPurgeable (IOSurfacePurgeabilityState newState, IntPtr oldStatePtr);
+
+		[Watch (4,0), TV (11,0), Mac (10,6), iOS (11,0), MacCatalyst (13,0)]
+		[Field ("kIOSurfaceColorSpace")]
+		NSString kIOSurfaceColorSpace { get; }
```
Comment (mandel-macaque) [inline src/iosurface.cs:234]: kIOSurfaceColorSpace is not correct. Those are global in objc and that is why they add the framework name and also that k which stands for key.

File: src/iosurface.cs [hunk 2]
```diff
@@ -228,5 +228,13 @@ interface IOSurface : NSSecureCoding
...
+
+		[Watch (4,0), TV (11,0), Mac (10,6), iOS (11,0), MacCatalyst (13,0)]
+		[Field ("kIOSurfaceICCProfile")]
+		NSString kIOSurfaceICCProfile { get; }
```
Comment (mandel-macaque) [inline src/iosurface.cs:238]: Same.

**Review by mandel-macaque — CHANGES_REQUESTED** [review]

**Review by rolfbjarne — CHANGES_REQUESTED** [review]
Just add the two new fields and P/Invokes to the common ignore file, there are a lot of other API that we haven't bound in this framework, and which we'll probably have to bind first for any of these to make sense.

**Review by rolfbjarne — APPROVED** [review]

**Comment by tj-devel709** [comment]
Unrelated Test Failure: https://github.com/xamarin/maccore/issues/2443

**Review by mandel-macaque — APPROVED** [review]

**Pull request merged**